examples/jsm/animation/CCDIKSolver.js

```diff
@@ -186,19 +186,13 @@ class CCDIKSolver {
 
 				if ( rotationMin !== undefined ) {
 
-					link.rotation.setFromVector3(
-						link.rotation
-							.toVector3( _vector )
-							.max( rotationMin ) );
+					link.rotation.setFromVector3( _vector.setFromEuler( link.rotation ).max( rotationMin ) );
```

Comment:
/ping @takahirox

1. There does not appear to be a test case for use of `rotationMin/Max`. Personally, I do not think it makes sense to clamp Euler angles anyway -- except in very special cases.

2. Also, but unrelated, I am not sure you want to call `updateMatrixWorld()` a few lines later. Maybe use `updateWorldMatrix()`, where you can control recursion?

Reply:
Thanks for pointing out. Let me take a look closer later...